### ISSUE TITLE
Support for getting the currency symbol directly.

### ISSUE THREAD

**Issue by rickvdl** [joshswan/react-native-globalize#40]
Hi,

Thanks for the library, it's working great 👍  . I was wondering whether there is an option to get the currency symbol directly so I can display it in other UI elements as well. For instance a settings screen showing the active currency or a label next to a monetary input field.

Thanks in advance,

Rick

**Comment by joshswan** [comment]
There isn't really a way of getting the symbol by itself at the moment, but I thought of a couple of hacks to get at it. First, you could use `withGlobalize` to get a currency formatter and strip the numbers:
```javascript
const currencyFormatter = this.props.globalize.getCurrencyFormatter(null, { minimumFractionDigits: 0, maximumFractionDigits: 0 });
const symbol = currencyFormatter(1).replace(/[0-9]/g, '');
```
Or (more hacky) you can access the stored runtime arguments of the currency formatter function:
```javascript
const symbol = this.props.globalize.getCurrencyFormatter().runtimeArgs[0].runtimeArgs[0][0];
```

**Comment by rickvdl** [comment]
Hi @joshswan,

Thanks for the quick reply, the first option will work for me for now. Would this be something that you could see being added to the library? If so I might give it a shot and see if I can come up with a more elegant solution, something like 
```javascript 
this.props.globalize.getCurrencySymbol(altNarrow: boolean): string
```

Thanks again, 

Rick

**Comment by joshswan** [comment]
Totally open to a PR for that if you want to take a stab at it. There's an open issue for adding this functionality to the underlying globalize library itself, which would obviously remove the need for hacky solutions, but it doesn't look like there's been any progress on it for the past year. So for now, this is probably the best solution.

**Comment by rickvdl** [comment]
@joshswan  Alright, I think I've come up with the most less-hacky way to achieve this in PR #41 . 

Let me know if you have any questions 😄

**Comment by joshswan** [comment]
Great stuff! That's a much more reliable way of handling it. Just published `v2.3.0` to npm.

**Comment by rickvdl** [comment]
@joshswan

Great man, thanks a lot!
